cPP_DSC.adoc

```diff
@@ -4019,203 +4015,391 @@ The dependencies between SFRs implemented by the TOE are addressed as follows.
 |Rationale Statement
 
 |FCS_CKM.1 
-|[FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_COP.1] 
+|[_FCS_CKM.1/AKG_, _FCS_CKM.1/SKG_, FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7, _FCS_CKM_EXT.8_ or FCS_COP.1] 
+
+FCS_CKM.3
 
 [FCS_RBG.1 or FCS_RNG.1]
 
-FCS_CKM.6 
-|FCS_CKM.2, FCS_CKM.6, FCS_CKM_EXT.7, FCS_COP.1 and FCS_RBG.1 are required by the PP.
+FCS_CKM.6
+|FCS_CKM.1/AKG - (selection-based) included
+
+FCS_CKM.1/SKG - (selection-based) included
+
+FCS_CKM.2 - included
+
+FCS_CKM.3 is satisfied by FAU_STG_EXT.1 which controls access to keys - included
```

Comment:
I guess we can't say FCS_CKM.3 is satisfied by FCS_CKM_EXT.3 because FCS_CKM_EXT.3 is selection-based?

Reply:
Sort of. If you look at what FCS_CKM.3 is talking about, it is access to the keys. In our cPP, FCS_STG_EXT.1 is what actually specifies how access control to the keys is handled, and one of the options from there is FCS_CKM_EXT.3, but also it could be hardware-controlled (as opposed to needing to be encrypted). So to me it seems like that was the actual better linkage in terms of how we meet the expectations for the CC:2022 dependencies.

Note that in almost all the crypto catalog they have removed FCS_CKM.3 as a dependency. I was adding it back because I didn't want to not list things that were in the CC:2022.